runtime/error.c

```diff
@@ -148,13 +165,66 @@ void _bal_error_backtrace_print(ErrorPtr ep) {
     fflush(stderr);
 }
 
+static void processPCs(GC PC* pcs, uint32_t nPCs, uint32_t start, int64_t lineNumber, BacktraceError *error) {
+    // Handle the starting pc seperately
+    processInitialPC(pcs[start], lineNumber, error);
+    if (error->code == NO_DEBUG_INFO) {
+        for (uint32_t i = start + 1; i < nPCs; i++) {
```

Comment:
Combine these two loops by having a function pointer that is backtrace_pcinfo or backtrace_syminfo.

Reply:
@jclark, Is that fine to use a function pointer because the function signatures of backtrace_pcinfo or backtrace_syminfo are different?
Maybe I have misunderstood your suggestion.

Reply:
I had not noticed they were different. My suggestion won't work.